src/main/java/org/opentripplanner/updater/stoptime/MqttGtfsRealtimeUpdater.java

```diff
@@ -0,0 +1,149 @@
+package org.opentripplanner.updater.stoptime;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.google.protobuf.InvalidProtocolBufferException;
+import com.google.transit.realtime.GtfsRealtime;
+import org.eclipse.paho.client.mqttv3.IMqttDeliveryToken;
+import org.eclipse.paho.client.mqttv3.MqttCallbackExtended;
+import org.eclipse.paho.client.mqttv3.MqttClient;
+import org.eclipse.paho.client.mqttv3.MqttConnectOptions;
+import org.eclipse.paho.client.mqttv3.MqttException;
+import org.eclipse.paho.client.mqttv3.MqttMessage;
+import org.eclipse.paho.client.mqttv3.persist.MemoryPersistence;
+import org.opentripplanner.routing.graph.Graph;
+import org.opentripplanner.updater.GraphUpdater;
+import org.opentripplanner.updater.GraphUpdaterManager;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * This class starts an Paho MQTT client which opens a connection to a GTFS-RT data source.
+ * A callback is registered which handles incoming GTFS-RT messages as they stream in by placing a
+ * GTFS-RT decoder Runnable task in the single-threaded executor for handling.
+ *
+ * Usage example ('bessersmith' name is an example) in the file 'Graph.properties':
+ *
+ * <pre>
+ * bessersmith.type = mqtt-gtfs-rt-updater
+ * bessersmith.feedId = hsl
+ * bessersmith.url = ssl://mqtt.hsl.fi:443
```

Comment:
@hannesj pointed out that the URL is currently different. However, this old URL given here should be production ready soon again. If I'm not mistaken, the new port for MQTT will be 8883 (as recently updated in the documentation [here](https://digitransit.fi/en/developers/apis/4-realtime-api/vehicle-positions/#api-endpoints))